### DISCUSSION TITLE
Can we get the ability to select the color of our ping in the party plugin?

### DISCUSSION THREAD

**Discussion by SandroVA** [runelite/runelite#15780] [Ideas]
Right now the color of my ping is orange and I am constantly being told it is difficult to see. Would be really nice if I could change it to any color I wanted.

**Comment by Ferrariic** [comment]
You can get a custom ping color for yourself with plugin-hub plugin NeverScapeAlone, a different party-like plugin. Iirc they don't have plans to incorporate custom colors in party default.
